src/Pages/form/Components/PatientInformationSection.tsx

```diff
@@ -176,27 +315,68 @@ const PatientInformationSection: React.FC = () => {
                   </Box>
                 </ToggleButton>
               </GenderStyledToggleButtonGroup>
-
               <Typography variant="body1">Female</Typography>
             </Box>
           </Box>
         </Grid>
       </Box>
       {/* Arrival Time & Onset */}
-      <ArrivalOnset />
+      <ArrivalOnset
+        values={values}
+        fieldName="PatientInformation"
+        onChange={onChange}
+      />
       {/* Upload CT Scan */}
       <Box className={classes.boxUpload}>
         <Box className={classes.textTitle}>
           <Typography variant="h4">Upload CT Scan</Typography>
+          <ErrorMessage name={`${fieldName}.file`}>
+            {(msg) => <Box className={classes.errorMessage}>{msg}</Box>}
+          </ErrorMessage>
         </Box>
         <Button
           variant="contained"
           component="label"
           className={classes.buttonUpload}
         >
           Upload
-          <input type="file" hidden />
+          <input
+            hidden
+            multiple
+            name="file"
+            type="file"
+            onChange={handleFileChange}
+          />
         </Button>
+        {showFileName && (
+          <Box>
+            <Grid container item style={{ margin: "8px" }} spacing={3}>
+              <Grid item>
+                <Typography>{values.file?.length} files</Typography>
+              </Grid>
+              <Grid item style={{ cursor: "pointer" }} onClick={removeAll}>
+                <Typography style={{ color: "#FF4181", textTransform: "none" }}>
+                  Remove all
+                </Typography>
+              </Grid>
+            </Grid>
+            <Box
+              height="150px"
+              overflow="auto"
+              borderRadius={20}
+              {...defaultProps}
```

Comment:
I'm not sure why you create defaulProps variable. Do you want to reuse that ? If not, you can move all to this line. If yes, I'm ok.